nbs/models.ipynb

```diff
@@ -57,6 +57,7 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.            self.model_['mean'] = mean
</pre><p>Let's remove this line, to avoid updating the object <code>self.model_</code> . Let's use <code>res</code> in <code>_calculate_intervals</code> instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
<p>Done. Also used res in the other models. </p>